core/lib/tasks/exchanges.rake

```diff
@@ -20,12 +20,13 @@ namespace :exchanges do
       begin
         item_charger.charge_for_items
       rescue Spree::UnreturnedItemCharger::ChargeFailure => e
-        failure = {message: e.message, new_order: e.new_order.try(:number)}
+        failure = { message: e.message }
       rescue Exception => e
```

Comment:
While you're here, if you could get rid of this, I'd really appreciate it.

```
rescue => e
```

Rescuing Exception will rescue syntax errors, signal exceptions and such. Things that you shouldn't be rescuing unless you're re-raising them.